packages/obelisk/obelisk.0.8.0/opam

```diff
@@ -0,0 +1,41 @@
+opam-version: "2.0"
+synopsis: "Pretty-printing for Menhir files"
+description: """
+Obelisk is a simple tool which produces pretty-printed output from a Menhir parser file (.mly).
+It is inspired from yacc2latex and is also written in OCaml, but is aimed at supporting features from Menhir instead of only those of ocamlyacc."""
+maintainer: ["Lélio Brun <lb@leliobrun.net>"]
+authors: ["Lélio Brun"]
+license: "MIT"
+homepage: "https://github.com/Lelio-Brun/Obelisk"
+doc: "https://github.com/Lelio-Brun/Obelisk/blob/master/README.md"
+bug-reports: "https://github.com/Lelio-Brun/obelisk/issues"
+depends: [
+  "ocaml" {>= "4.08"}
+  "dune" {>= "2.2.0"}
+  "menhir" {>= "20190613"}
+  "re"
+]
+build: [
+  ["dune" "subst"] {pinned}
```

Comment:
```suggestion
  ["dune" "subst"] {dev}
```

See https://github.com/ocaml/dune/pull/3647 for details. If you update your dune version dependency, the correct package should be generated in the future.